tiledb/sm/query/query.cc

```diff
@@ -2578,6 +2607,7 @@ Status Query::set_subarray(const void* subarray) {
     // the fragment for global order writes. We need to prevent this, thus
     // persist the fragment between remote global order write submits.
     if (strategy_ != nullptr && layout_ != Layout::GLOBAL_ORDER) {
+      // TODO Create function that does this.
```

Comment:
Should this be there?

Reply:
Nope, that is a comment I forgot to remove. This whole function is a mess. The reset called here does nothing: reset is a nullop for all write queries *except* global order.